### ISSUE TITLE
ImageNet64, classifier guidance ADM-G is worse than ADM regarding FID, but better than ADM regarding IS

### ISSUE THREAD

**Issue by forever208** [openai/guided-diffusion#119]
Hi, can anybody share the FID of ADM-G on ImageNet 64?
I get a worse FID than the unconditional ADM

**Comment by forever208** [comment]
Hi @unixpickle , thanks for the work and code.

with the pre-trained model, I tested the conditional ADM-G and ADM on ImageNet64 with 100 sampling steps and dropout 0.1.
**ADM gets FID 3.25** (looks normal)

but the FID for ADM-G is strange:
**ADM-G scale 0.3 = 4.14 FID
ADM-G scale 1 = 7.22 FID
ADM-G scale 3 = 13.67 FID**

But ADM-G achieves higher IS than ADM.

Is this normal in your experiments?

Thank you very much!
